lua/lls.lua

```diff
@@ -0,0 +1,23 @@
+local gatekeeperc = require("gatekeeperc")
+
+local M = {}
+
+-- Function that sets up the GK functional block.
+function M.setup_block(net_conf)
+
+	-- Init the LLS configuration structure.
+	local lls_conf = gatekeeperc.get_lls_conf()
+	if lls_conf == nil then return nil end
+
+	-- Change these parameters to configure the LLS block.
+	lls_conf.lcore_id = 4
```

Comment:
I just pushed a patch to make the allocation of a single lcore simpler. You'll need to pull & rebase to see it.